rules-reviewed/eap6/jboss-eap5/jboss-eap5-xml.windup.xml

```diff
@@ -245,7 +245,6 @@
                     <description>
                         The `jbosscmp-jdbc.xml` is a deployment decriptor controlling the Container Managed Persistence (CMP).
                     </description>
-                    <link title="JBoss EAP 4: The jbosscmp-jdbc Structure" href="https://access.redhat.com/documentation/en-us/jboss_enterprise_application_platform/4.3/html-single/server_configuration_guide/#The_CMP_Engine-The_jbosscmp_jdbc_Structure"/>
```

Comment:
We won't have any documentation link in this rule. Well, if there is no way to link this rule to an EAP 5 doc.....

Reply:
We can link to EAP 5. It's EAP 4 that is gone. I will look for a suitable link.

Reply:
Found the link for `jbosscmp-jdbc.xml` in EAP 5 docs. Will update.